Analyses/test/bfieldPlotter.fcl

```diff
@@ -0,0 +1,32 @@
+# -*- mode:tcl -*-
+# Example of running BFieldPlotter
+#
+
+#include "JobConfig/primary/flateminus.fcl"
```

Comment:
You don't need this include since you don't need generate or g4run

Reply:
I just realized why you have this line.  You are pulling in some very deeply nested includes.

What you need is:

#include "fcl/minimalMessageService.fcl"
#include "fcl/standardServices.fcl"
services: @local::Services.Core

This defines a little more than you need for this job but it's not a lot more and it's the smallest chunk that's robust against future evolution of our fcl configuration (ie we will always define Services.Core and it will always have a properly configured GeoemtryService ).

Reply:
I had done it that way since it was simple to get the test script going, but I agree the better test file is one simply using the relevant services and less sensitive to outside changes. I've made this switch!